### ISSUE TITLE
Please consider  bring  an complete examples use ammer 

### ISSUE THREAD

**Issue by sonygod** [Aurel300/ammer#16]
[wish librarys]Please consider  bring a complete example use ammer 

current Hashlink has Libuv socket framework, but there is no WebSocket framework.

Maybe consider importing **uWebSockets** or  **Libwebsockets** to showcase what ammer can do.

**Comment by Aurel300** [comment]
Yes, it is planned. Both of the libraries you mention might be quite heavy-duty to start off with. I think wrapping Lua with ammer (for a Lua runtime in any ammer target) might be the first step, since the interface is quite minimal. A new SteamWrap is also planned.
